### PR TITLE
Status card bug

### PR DESCRIPTION
**Pull request by upatricck**
Deleting one sesh site will delete associated status card.
But deleting multiple sesh_site with no delete the associated status card.

This is because deleting multiple items does not call the delete method of each instance but rather the one for the manage.

## To be done
* Overwritting the delete of the QuerySet

### CODE REVIEW

File: seshdash/models.py [hunk 1]
```diff
@@ -136,11 +136,12 @@ def save(self, *args, **kwargs):
         else:
             super(Sesh_Site, self).save(*args, **kwargs)
 
-    def delete(self):
+    def delete(self, *args, **kwargs):
         # Delete the site and its associated status card
         status_card = self.status_card
+        print "Deleting associated status card"
```
Comment (jlark) [inline seshdash/models.py]: can we stop using prints and always use the logger? This goes for all the other changes. Not sure if the ones in the view.py are yours

**Comment by upatricck** [comment]
Aaahhhh always forgets, fixing it.

File: seshdash/views.py [hunk 2]
```diff
@@ -156,13 +157,19 @@ def import_site(request):
     """
     Initial login to gather VRM and Account information
     """
+    print "In the import site "
     context_dict = {}
     template = 'seshdash/initial-login.html'
     if request.method == "POST":
+            print "The request is post"
```
Comment (jlark) [inline seshdash/views.py]: all these prints?

File: seshdash/views.py [hunk 3]
```diff
@@ -207,19 +214,25 @@ def import_site(request):
 
                     context_dict['site_forms'] = site_forms_factory(initial = pre_pop_data,
                                                                     instance=VRM)
+
+                    print "The site forms are: ",
```
Comment (jlark) [inline seshdash/views.py]: here as well? too many prints.

**Comment by jlark** [comment]
LGTM

**Pull request merged**